build.sbt

```diff
@@ -92,7 +93,8 @@ lazy val coreDeps = commonDeps ++ Seq(
   "org.scodec"           %% "scodec-bits"       % "1.0.10",
   "org.scalactic"        %% "scalactic"         % "2.2.6",
   "com.markatta"         %% "futiles"           % "1.1.3",
-  "com.nativelibs4java"  %% "scalaxy-loops"     % "0.3.3" % "provided"
+  "com.nativelibs4java"  %% "scalaxy-loops"     % "0.3.3" % "provided",
+  "com.github.scala-incubator.io" %% "scala-io-file" % "0.4.2"
```

Comment:
Is this the repo?  Can you point me to the latest repo?
https://github.com/scala-incubator/scala-io